dissect/target/plugins/os/unix/log/helpers.py

```diff
@@ -0,0 +1,44 @@
+import itertools
+import logging
+import re
+from datetime import datetime
+from typing import Iterator
+
+from dissect.target.helpers.fsutil import TargetPath, open_decompress
+
+log = logging.getLogger(__name__)
+
+_RE_TS = r"^[A-Za-z]{3}\s*[0-9]{1,2}\s[0-9]{1,2}:[0-9]{2}:[0-9]{2}"
+_RE_TS_ISO = r"^\d{4}-\d{2}-\d{2}T\d{2}:\d{2}:\d{2}\.\d{6}\+\d{2}:\d{2}"
+
+RE_TS = re.compile(_RE_TS)
+RE_TS_ISO = re.compile(_RE_TS_ISO)
+RE_LINE = re.compile(
+    rf"(?P<ts>{_RE_TS}|{_RE_TS_ISO})\s(?P<hostname>\S+)\s(?P<service>\S+?)(\[(?P<pid>\d+)\])?:\s(?P<message>.+)$"
```

Comment:
Some small changes to make the code below more compact:

- Start regex with matching on the similar ending of both timestamps
- Do match on the hostname, but do not capture it since it's not used
- Rename service to daemon, such that it matches the Record Descriptor
- Strip spaces left and right for the message
- Make the regex verbose to be able to provide comments

```suggestion
   RE_LINE = re.compile(
    r"""
    \d{2}:\d{2}\s                           # First match on the similar ending of the different timestamps
    (?:\S+)\s                               # The hostname, but do not capture it
    (?P<daemon>\S+?)(\[(?P<pid>\d+)\])?:    # The daemon with optionally the PID between brackets
    \s*(?P<message>.+?)\s*$                 # The log message stripped from spaces left and right
    """,
    re.VERBOSE,
)
```

and make sure you use `RE_LINE.search(line)` instead of `RE_LINE.match(line)` in `messages.py`